UnivaPay-for-WooCommerce/composer.json

```diff
@@ -20,11 +19,11 @@
             "wp plugin install woocommerce --activate --path=/tmp/wordpress",
             "cp -r . /tmp/wordpress/wp-content/plugins/UnivaPay-for-WooCommerce",
             "rm -rf /tmp/wordpress/wp-content/plugins/UnivaPay-for-WooCommerce/vendor",
-            "phpunit"
+            "WP_ENV=test phpunit"
         ]
     },
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.4",
```

Comment:
Is this aligned with the min version for woo commerce?

Reply:
@kgston I am assuming that we are upgrading the PHP SDK to the latest version compatible with PHP 7.4. With the current automated test setup, the oldest WooCommerce version I can support is 8.2. [WC doc](https://woocommerce.com/document/update-php-wordpress/).

I would like to add support for older versions, but I will need to set up the tests differently to achieve that, as wp-cli does not provide older WooCommerce versions (afaik).

Reply:
@kgston I am assuming that we are upgrading the PHP SDK to the latest version compatible with PHP 7.4. With the current automated test setup, the oldest WooCommerce version I can support is 8.2. [WC doc](https://woocommerce.com/document/update-php-wordpress/).

I would like to add support for older versions, but I will need to set up the tests differently to achieve that, as wp-cli does not provide older WooCommerce versions (afaik).

Reply:
and here is the current [test matrix](https://github.com/univapay/UnivaPay-for-WooCommerce/blob/main/.github/workflows/main.yml#L13-L16)

Reply:
in the link that i provide for WooCommerce’s doc on section 1 do mention this one
`PHP versions less than 8.1 and MySQL versions less than 5.6 no longer receive active support, many versions are at End Of Life, and are therefore no longer maintained. As such, using outdated and unsupported versions of MySQL and PHP may expose your site to security vulnerabilities. `

Reply:
All right then 👌